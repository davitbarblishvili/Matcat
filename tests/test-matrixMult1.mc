// Author: Davit
func main() int {
    matrix a; 
    matrix c; 

    a = [[1,2][4,5]];
   

    c = a * a;
    printm(c);

    return 0;
}