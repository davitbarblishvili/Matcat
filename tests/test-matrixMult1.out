(
[9.00 12.00 ]
[24.00 33.00 ]
)